src/style/index.css

```diff
@@ -0,0 +1,11 @@
+/* abstracts */
+@import url(./abstracts/variables.css);
+
+
+/* bases */
+@import url(./bases/reset.css);
```

Comment:
Please create index.css file in the bases folder and then import global.css, reset.css, and typography.css file into it